classes/fields/datetime.php

```diff
@@ -831,4 +831,96 @@ public function convert_format( $source_format, $args = array() ) {
 		}
 		return $new_format;
 	}
+
+	/**
+	 * Enqueue the i18n files for jquery date/timepicker
+	 *
+	 * @since  2.7
+	 */
+	public function enqueue_jquery_ui_i18n() {
+		static $done = array();
+
+		$locale = $this->get_locale_jquery_ui_i18n();
+
+		if ( ! empty( $locale ) ) {
+
+			$types = array();
+			switch ( static::$type ) {
+				case 'time':
+					$types[] = 'time';
+				break;
+				case 'date':
+					$types[] = 'date';
+				break;
+				case 'datetime':
+					$types[] = 'time';
+					$types[] = 'date';
+				break;
+			}
+
+			if ( in_array( 'date', $types, true ) &&
+			     ! in_array( 'date', $done, true ) &&
+			     function_exists( 'wp_localize_jquery_ui_datepicker' )
+			) {
+				wp_localize_jquery_ui_datepicker();
+				$done[] = 'date';
+			}
+
+			if ( in_array( 'time', $types, true ) && ! in_array( 'time', $done, true ) ) {
+
+				// Local files.
+				if ( ! file_exists( PODS_DIR . 'ui/js/timepicker/i18n/jquery-ui-timepicker-' . $locale . '.js' ) ) {
```

Comment:
@sc0ttkclark Because of this file_existence check we might even want to remove the whole `get_locale_jquery_ui_i18n()` method. Not sure how this would impact performance though..